config/checker/org.eclipse.jgit.astub

```diff
@@ -0,0 +1,14 @@
+
+import org.checkerframework.checker.nullness.qual.NonNull;
+import org.checkerframework.common.aliasing.qual.NonLeaked;
+import org.checkerframework.common.aliasing.qual.Unique;
+
+package org.eclipse.jgit.revwalk;
+
+class RevWalk {
```

Comment:
This is awesome :1st_place_medal:

Reply:
In the super-ideal world, each value in the program should be immutable; in an almost-super-ideal world where we need to work with external mutable APIs, each value that's of a mutable type should at least be `@Unique` :unicorn: